time stamp
string manager_prefix
